packages/reaction-core/server/methods/orders.js

```diff
@@ -56,24 +56,25 @@ Meteor.methods({
     check(order, Object);
     check(shipment, Object);
     check(packed, Boolean);
-    this.unblock();
```

Comment:
I removed this because we have one more internally called method here. And it is still not 100% clear to me how `this.unblock()` could affect in such situation, so I believe it is safer to remove it. Anyway it is small method. Here is nothing to unblock from my point of view,